Tests/SwiftIDEUtilsTest/Assertions.swift

```diff
@@ -0,0 +1,97 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import SwiftIDEUtils
+import SwiftParser
+import SwiftSyntax
+import _SwiftSyntaxTestSupport
+
+/// Parse `source` and checks its `classifications` is the same as `expected`. The `classification` filter out all the ``Trivia``s in `source` i.e. ``SyntaxClassification`` is `none` to make test cases look nicer.
```

Comment:
We aren’t actually filtering out the trivia. We do have classifications for comments. We are just not including any `none` classifications.
```suggestion
/// Parse `source` and checks its `classifications` is the same as `expected`. 
/// 
/// The `expected` classifications should only contain classifications that are not `none`. All uncovered ranges are expected to have no classification.
```